misk/src/test/kotlin/misk/web/actions/DashboardMetadataActionTest.kt

```diff
@@ -63,6 +63,28 @@ class DashboardMetadataActionTest {
     assertNotNull(response.tabs.find { it.name == "Test Dashboard Tab" })
   }
 
+  @Test fun loadDashboardNavbarItems() {
+    val dashboard = DashboardMetadataActionTestDashboard::class.simpleName!!
+    val response = executeRequest(
+      path = path + dashboard,
+      user = "sandy",
+      capabilities = "test_admin_access"
+    )
+    assertEquals(1, response.navbar_items.size)
+    assertNotNull(response.navbar_items.find { it.contains("Test Navbar Link") })
```

Comment:
Would be nice to use a non-empty href and assert that the anchor tag is formed as expected with the correct href

Reply:
I'll update it to a full `assertEquals` snapshot since the main test is that the bound string is present, not necessarily that it is a `<a>` link.